isaric/parsers/ncov-phosp.toml

```diff
@@ -0,0 +1,803 @@
+      { field = "crf1a_o2_blniv", ref = "Y/N" },
+      { field = "crf1a_o2_cpapv", ref = "Y/N" },
+    ]
+
+  [visit.treatment_invasive_ventilation]
+    field = "crf1a_o2_imv"
+    description = "e) Invasive Mechanical Ventilation"
+    ref = "Y/N"
+
+  [visit.treatment_antibiotics]
+    field = "crf1a_treat_at"
+    description = "a) Antibiotic therapy"
+    ref = "Y/N"
+
+  [visit.treatment_noninvasive_ventilation]
+    field = "crf1a_o2_blniv"
+    description = "c) Bi-Level Non-Invasive Ventilation"
+    ref = "Y/N"
+
+  [visit.treatment_high_flow_nasal_cannula]
+    field = "crf1a_o2_hfn"
+    description = "d) High Flow Nasal O2"
+    ref = "Y/N"
+
+  [visit.treatment_immunosuppressant]
+    combinedType = "any"
+    fields = [
+      { field = "crf1a_treat_saril", ref = "Y/N" },
+      { field = "crf1a_treat_tocil", ref = "Y/N" },
+    ]
+
+  [visit.treatment_prone_position]
+    field = "crf1a_treat_proning"
+    ref = "Y/N"
+
+  [visit.treatment_anticoagulation]
+    field = "crf1a_treat_tdac"
+    ref = "Y/N"
+
+  [visit.treatment_cardiovascular_support]
+    field = "crf1a_o2_ecmo"
+    ref = "Y/N"
+
+  [visit.treatment_monoclonal_antibody]
+    field = "crf1a_treat_saril"
+    ref = "Y/N"
+
+  [visit.treatment_antivirals]
+    field = "crf1a_treat_remde"
+    ref = "Y/N"
+
+  [visit.date_outcome]
+    combinedType = "firstNonNull"
+    fields = [
+      { field = "crf1a_date_dis", description = "Date of discharge from hospital", if = { crf1a_date_dis = { "!=" = "NA" } } },
+      { field = "crf2a_date_discharged", description = "Date of discharge from hospital", if = { crf2a_date_discharged = { "!=" = "NA" } } },
+      # { field = "crfev_date_discharge", description = "Date of discharge" },
+    ]
+
+    # OBSERVATION
+    # required fields - phase, date, name
+
+[[observation]]
+  name = "diastolic_blood_pressure_mmHg"
+  date = { field = "crf3a_visit_date" }
+  phase = "followup"
+  value = { field = "crf3a_rest_bp_dia" }
+  if = { crf3a_rest_bp_dia = { "!=" = "NA" } }
+
+[[observation]]
+  name = "diastolic_blood_pressure_mmHg"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  value = { field = "crf1a_bp_dia" }
+  if = { crf1a_bp_dia = { "!=" = "NA" } }
+
+[[observation]]
+  name = "diastolic_blood_pressure_mmHg"
+  date = { field = "crf3b_date_visit" }
+  phase = "followup"
+  value = { field = "crf3b_rest_bp_dia" }
+  if = { crf3b_rest_bp_dia = { "!=" = "NA" } }
+
+[[observation]]
+  name = "diastolic_blood_pressure_mmHg"
+  date = { field = "crf4a_visit_date" }
+  phase = "followup"
+  value = { field = "crf4a_rest_bp_dia" }
+  if = { crf4a_rest_bp_dia = { "!=" = "NA" } }
+
+[[observation]]
+  name = "abdominal_pain"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  is_present = { field = "crf1a_pain", ref = "Y/N" }
+
+[[observation]]
+  name = "abdominal_pain"
+  date = { field = "patient_sq_date" }
+  phase = "followup"
+  is_present = { field = "abdominal_pain", ref = "Y/N" }
```

Comment:
@ekamau there is also stomach_pain captured in the PSQ which could be combined with this attribute?

Reply:
noted!